docs/template_plugin/tests/functional/op_reference/ctc_greedy_decoder_seq_len.cpp

```diff
@@ -0,0 +1,228 @@
+    Exec();
+}
+
+TEST_P(ReferenceCTCGreedyDecoderSeqLenTestNoOptionalInput, CompareWithRefs) {
+    Exec();
+}
+
+template <element::Type_t ET>
+std::vector<CTCGreedyDecoderSeqLenParams> generateParams() {
+    using T = typename element_type_traits<ET>::value_type;
+    std::vector<CTCGreedyDecoderSeqLenParams> params {
+        CTCGreedyDecoderSeqLenParams(
+            Tensor(ET, {1, 3, 3}, std::vector<T>{0.1f, 0.2f, 0.f, 0.4f, 0.3f, 0.f, 0.5f, 0.6f, 0.f}),
+            Tensor(element::i32, {1}, std::vector<int32_t>{2}),
+            Tensor(element::i32, {}, std::vector<int32_t>{2}),
+            false,
+            Tensor(element::i32, {1, 3}, std::vector<int32_t>{1, 0, -1}),
+            Tensor(element::i32, {1}, std::vector<int32_t>{2}),
+            "evaluate_ctc_greedy_decoder_seq_len"),
+        CTCGreedyDecoderSeqLenParams(
+            Tensor(ET, {1, 3, 3}, std::vector<T>{0.1f, 0.2f, 0.f, 0.4f, 0.3f, 0.f, 0.5f, 0.6f, 0.f}),
+            Tensor(element::i32, {1}, std::vector<int32_t>{2}),
+            Tensor(element::i32, {}, std::vector<int32_t>{2}),
+            true,
+            Tensor(element::i32, {1, 3}, std::vector<int32_t>{1, 0, -1}),
+            Tensor(element::i32, {1}, std::vector<int32_t>{2}),
+            "evaluate_ctc_greedy_decoder_seq_len_merge"),
+        CTCGreedyDecoderSeqLenParams(
+            Tensor(ET, {2, 3, 3}, std::vector<T>{0.1f,  0.2f,  0.f, 0.15f, 0.25f, 0.f, 0.4f,  0.3f,  0.f,
+                                                 0.45f, 0.35f, 0.f, 0.5f,  0.6f,  0.f, 0.55f, 0.65f, 0.f}),
+            Tensor(element::i32, {2}, std::vector<int32_t>{1, 1}),
+            Tensor(element::i32, {}, std::vector<int32_t>{2}),
+            false,
+            Tensor(element::i32, {2, 3}, std::vector<int32_t>{1, -1, -1, 0, -1, -1}),
+            Tensor(element::i32, {2}, std::vector<int32_t>{1, 1}),
+            "evaluate_ctc_greedy_decoder_seq_len_multiple_batches"),
+        CTCGreedyDecoderSeqLenParams(
+            Tensor(ET, {3, 3, 3}, std::vector<T>{0.1f,  0.2f,  0.f, 0.15f, 0.25f, 0.f, 0.4f,  0.3f,  0.f,
+                                                 0.45f, 0.35f, 0.f, 0.5f,  0.6f,  0.f, 0.55f, 0.65f, 0.f,
+                                                 0.1f,  0.2f,  0.f, 0.15f, 0.25f, 0.f, 0.4f,  0.3f,  0.f}),
+            Tensor(element::i32, {3}, std::vector<int32_t>{2, 3, 1}),
+            Tensor(element::i32, {}, std::vector<int32_t>{2}),
+            false,
+            Tensor(element::i32, {3, 3}, std::vector<int32_t>{1, 1, -1, 0, 1, 1, 1, -1, -1}),
+            Tensor(element::i32, {3}, std::vector<int32_t>{2, 3, 1}),
+            "evaluate_ctc_greedy_decoder_seq_len_multiple_batches2"),
+    };
+    return params;
+}
+
+std::vector<CTCGreedyDecoderSeqLenParams> generateCombinedParams() {
+    const std::vector<std::vector<CTCGreedyDecoderSeqLenParams>> generatedParams {
+        generateParams<element::Type_t::bf16>(),
+        generateParams<element::Type_t::f16>(),
```

Comment:
the same

Reply:
In https://docs.openvino.ai/latest/openvino_docs_ops_sequence_CTCGreedyDecoderSeqLen_6.html, input type is floating point.